src/RockBridge.cpp

```diff
@@ -1,193 +0,0 @@
-	
-	// Import typekits to allow RTT convert the types used by the components
-	RTT::types::TypekitRepository::Import(new orogen_typekits::stdTypekitPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::stdCorbaTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::stdMQueueTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::stdTypelibTransportPlugin);
-	
-	RTT::types::TypekitRepository::Import(new orogen_typekits::baseTypekitPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::baseCorbaTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::baseMQueueTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::baseTypelibTransportPlugin);
-	
-	RTT::types::TypekitRepository::Import(new orogen_typekits::gazeboTypekitPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::gazeboCorbaTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::gazeboMQueueTransportPlugin);
-	RTT::types::TypekitRepository::Import(new orogen_typekits::gazeboTypelibTransportPlugin);
-	
-		
-	// Each simulation step the Update method is called to update the simulated sensors and actuators
-	eventHandler.push_back(
-			event::Events::ConnectWorldUpdateBegin(
-					boost::bind(&RockBridge::updateBegin,this, _1)));
-	eventHandler.push_back(
-			event::Events::ConnectWorldUpdateEnd(
-					boost::bind(&RockBridge::updateEnd,this)));
-	eventHandler.push_back(
-			event::Events::ConnectWorldCreated(
-					boost::bind(&RockBridge::worldCreated,this, _1)));
-	eventHandler.push_back(
-			event::Events::ConnectAddEntity(
-					boost::bind(&RockBridge::modelAdded,this, _1)));
-}
-//======================================================================================
-// worldCreated() is called every time a world is added
-void RockBridge::worldCreated(std::string const& worldName)
-{
-    gzmsg << "RockBridge: initializing world: " << worldName << std::endl;
-
-	physics::WorldPtr world = physics::get_world(worldName);
-    if (!world)
-    {
-            gzerr << "RockBridge: cannot find world " << worldName << std::endl;
-            return;
-    }
-
-	int environment = GROUND;
-	std::string world_name("underwater");
-	if(world_name.compare((world)->GetName()) == 0)
-	{
-		environment = UNDERWATER; 
-		gzmsg << "RockBridge: underwater world detected. " << std::endl;
-	}	
-		    
-    typedef physics::Model_V Model_V;
-    Model_V model_list = world->GetModels();
-	for(Model_V::iterator model_it = model_list.begin(); model_it != model_list.end(); ++model_it)
-	{
-		createTask(world,*model_it,environment);
-	}
-	model_list.clear();
-	worlds.push_back(world);
-}
-//======================================================================================
-void RockBridge::modelAdded(std::string const& modelName)
-{
-	gzmsg << "RockBridge: added entity: " << modelName << std::endl;
```

Comment:
This is dead code, delete it. Git is here if you need to get it back sometimes in the future.